Startup.cs

```diff
@@ -0,0 +1,72 @@
+using System;

```

Comment:
Pridėk į repozitoriją ne tik šitą vieną failiuką, bet visus failus, kurie nėra bin foldery.
Iš esmės kad visi kas pasiimtų tavo sucommitintą kodą, galėtų atsidaryt .sln failiuką su visual studio ir pasileist tavo webappsą.